src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -121,6 +148,26 @@ export class QuickBar extends LitElement {
     `;
   });
 
+  private _handleInputKeyDown(ev: KeyboardEvent) {
+    if (ev.code === "Enter") {
+      if (this._commandMode) {
+        this._runCommandAndCloseDialog(this._topCommandResult);
+      } else {
+        this._launchMoreInfoDialog(this._topEntityIdResult);
+      }
+    } else if (ev.code === "ArrowDown") {
+      ev.preventDefault();
+      this._firstListItem?.focus();
+    }
+  }
+
+  private _handleListItemKeyDown(ev: KeyboardEvent) {
+    const isSingleCharacter = ev.key.length === 1;
+    if (ev.code === "ArrowUp" || isSingleCharacter) {
```

Comment:
This prevents running the code when the first argument is true
```suggestion
    if (ev.code === "ArrowUp" || ev.key.length === 1) {
```

Reply:
It's so that when you arrow down into the list, you can arrow up to go back into the field.

It was originally on just the first list item, but I guess that changed so it's now focusing no matter where you are in the list. 

What's funny is that it just _happens_ to still work as expected even when you're mid-list because we don't prevent default. It focuses into the input field then immediately goes back into the list.

I'll find a way to have it only happen with the first list item

Reply:
Done